File: cloud/oauth-server/src/main/java/pers/acp/admin/oauth/route/NameUtils.java
package pers.acp.admin.oauth.route;

/**
 * @author zhang by 17/03/2019
 * @since JDK 11
 */
class NameUtils {
    private static final String GENERATED_NAME_PREFIX = "_genkey_";

    static String generateName(int i) {
        return GENERATED_NAME_PREFIX + i;
    }

}

File: cloud/route-server/src/main/kotlin/pers/acp/admin/route/definition/NameUtils.kt
package pers.acp.admin.route.definition

/**
 * @author zhang by 17/03/2019
 * @since JDK 11
 */
internal object NameUtils {

    private const val GENERATED_NAME_PREFIX = "_genkey_"

    fun generateName(i: Int): String {
        return GENERATED_NAME_PREFIX + i
    }

}
